nemo/nemo/utils/exp_logging.py

```diff
@@ -5,8 +5,15 @@
 from shutil import copyfile
 import subprocess
 import sys
+import warnings
 
-loggers = {}
+import nemo
+
+
+def get_logger(unused):
```

Comment:
third definition :]

Reply:
Yea, we kinda had three redundant definitions before, but now we only have one logging object!

Reply:
nice! :)